code/datums/station_traits/negative_traits.dm

```diff
@@ -134,3 +134,85 @@
 		/// The bot's language holder - so we can randomize and change their language
 		var/datum/language_holder/bot_languages = found_bot.get_language_holder()
 		bot_languages.selected_language = bot_languages.get_random_spoken_language()
+
+/datum/station_trait/revenge_of_pun_pun
+	name = "Revenge of Pun Pun"
+	trait_type = STATION_TRAIT_NEGATIVE
+	weight = 2
+	var/static/list/weapon_types
+
+/datum/station_trait/revenge_of_pun_pun/New()
+	if(!weapon_types)
+		weapon_types = list(
+			/obj/item/chair = 20,
+			/obj/item/tailclub = 10,
+			/obj/item/melee/chainofcommand/tailwhip = 10,
+			/obj/item/melee/chainofcommand/tailwhip/kitty = 10,
+			/obj/item/reagent_containers/food/drinks/bottle = 20,
+			/obj/item/reagent_containers/food/drinks/bottle/kong = 5,
+			/obj/item/switchblade = 10,
+			/obj/item/sign/random = 10,
+			/obj/item/gun/ballistic/automatic/pistol = 1,
```

Comment:
but monkys cant shoot guns

Reply:
They're supposed to be able to but a recent change broke that aparently

Reply:
IMO, the chance for a gun to be chosen here should be pretty darn high, seeing as how the joke being referenced here is a monkey with a gun

Reply:
I'd also recommend choosing a gun with a decent melee force as the one you give him, in case he runs out of ammo

Reply:
> 
> 
> IMO, the chance for a gun to be chosen here should be pretty darn high, seeing as how the joke being referenced here is a monkey with a gun

No

Reply:
> They're supposed to be able to but a recent change broke that aparently

the joke falls flat if they cant fire it though so you should fix that

Reply:
I'm in the middle of another pr to fix that right now in addition to greatly increasing their smarts in a bunch of other ways